.travis.yml

```diff
@@ -34,10 +34,10 @@ jobs:
         - bash .run_chisel_tests.sh
     - stage: test
       script:
-        - ./.run_formal_checks.sh RocketCore
+        - travis_wait ./.run_formal_checks.sh RocketCore
```

Comment:
what does this do?

Reply:
It's supposed to increase the time before travis kills a test due to "timeout". It seems that this is a bigger problem with Yosys not being able to handle this change so I'm looking into it.